src/probability/independence.lean

```diff
@@ -733,4 +871,173 @@ hf_Indep.indep_fun_finset_prod_of_not_mem hf_meas finset.not_mem_range_self
 
 end indep_fun
 
+
+/-! ### Kolmogorov's 0-1 law
+
+Let `s : ι → measurable_space Ω` be an independent sequence of sub-σ-algebras. Then any set which
+is measurable with respect to the tail σ-algebra `limsup at_top s` has probability 0 or 1.
+-/
+
+section zero_one_law
+
+variables {m m0 : measurable_space Ω} {μ : measure Ω}
+
+lemma measure_eq_zero_or_one_or_top_of_indep_set_self {t : set Ω} (h_indep : indep_set t t μ) :
+  μ t = 0 ∨ μ t = 1 ∨ μ t = ⊤ :=
```

Comment:
can you use \ infty instead of \top here and elsewhere?

Reply:
Yes of course. I am surprised that I am still using top myself after having written that same review on several PRs :)